components/integrations/installed_incoming_webhook.tsx

```diff
@@ -74,11 +80,38 @@ type Props = {
     channel: Channel;
 }
 
-export default class InstalledIncomingWebhook extends React.PureComponent<Props> {
+type State = {
+    hovering: boolean;
+}
```

Comment:
changing this via state is quite render-expensive.
Please refactor using the `:hover` selector and hide elements that should not be visible when not hovering and vice versa